src/main/java/org/paymentsds/mpesa/Request.java

```diff
@@ -72,7 +72,11 @@ public Builder amount(double amount) {
         }
 
         public Builder from(String from) {
-            this.from = from;
+            if (from.length() == 9) {
+                this.from = 258 + from;
```

Comment:
Since this field is a String, let's use a String literal here
```suggestion
                this.from = "258" + from;
```